nixos/modules/services/networking/tinc.nix

```diff
@@ -383,9 +383,9 @@ in
         };
         preStart = ''
           mkdir -p /etc/tinc/${network}/hosts
-          chown tinc.${network} /etc/tinc/${network}/hosts
+          chown tinc:${network} /etc/tinc/${network}/hosts
           mkdir -p /etc/tinc/${network}/invitations
-          chown tinc.${network} /etc/tinc/${network}/invitations
+          chown tinc:${network} /etc/tinc/${network}/invitations
```

Comment:
This is not correct. The group is actually called tinc.${network}.

Reply:
Sorry about that, I will make a PR